### ISSUE TITLE
VST Scaling?

### ISSUE THREAD

**Issue by whannel** [ValdemarOrn/CloudSeed#3]
Not sure if this supports VST Scaling or if it something that could be requested please?

For high DPI display (2736x1824 with weird 1.5 ratio) it doesn't load displaying all controls (maybe a quarter or so get cropped) in FL20 with Windows Display settings Scale set to 150%, though as a workaround I found setting Windows to 100%, launching FL and loading CloudSeed I can then reset to Windows 150% - it would be nice if this wasn't needed.

Otherwise brilliant plugin, love it, be cool if a shimmer could be added, manual ways of doing that sounded a little tricky for me.

**Comment by tezza1971** [comment]
Yeah still having the same experience on Windows 10 Studio One 5 using a UHD resolution display in 2020

**Comment by neworldemancer** [comment]
Similar on 4k screen: when i open it - affects scaling in ableton in a horible manner, and yet doesn't display all own controls.
